### PR TITLE
Feature/partner page

### PR DESCRIPTION
**Pull request by hectoruch**
Update styles and create sections on partner page:

- Partner section
- Funders section
- Researchs section
- Other funders section

Add logos/images

### CODE REVIEW

File: app/src/components/Partners.jsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import CoverPrimary from './Shared/CoverPrimary';
 import Footer from './Shared/Footer';
+import PartnerSection from './Partners/PartnerSection'
```
Comment (andresgnlez) [inline app/src/components/Partners.jsx]: add `;` at the end of the line.

File: app/styles/components/c-partner-section.scss [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+    li {
+      display: inline-block;
+      min-width: 340px;
+      padding-right: 22px;
+
+      p {
+        font-size: $font-size-small;
+        line-height: 1.6;
+      }
```
Comment (andresgnlez) [inline app/styles/components/c-partner-section.scss]: This paragraph styles maybe are not necessary. Try removing them. We are using global ones with same properties.

Comment (hectoruch) [inline app/styles/components/c-partner-section.scss]: I remove only line-height property, but font-size is neccesary, that is smaller than the size we have on base styles.

File: app/src/components/Partners/PartnerSection.jsx [hunk 3]
```diff
@@ -20,7 +20,8 @@ class PartnerSection extends Component {
             third of the world’s wild fish catch. With over 100 victories that stop overfishing, habitat destruction,
             pollution and killing of threatened species like turtles and sharks, Oceana’s campaigns are delivering
             results. A restored ocean means that one billion people can enjoy a healthy seafood meal, every day,
-            forever. Together, we can save the oceans and help feed the world. Visit www.oceana.org to learn more.
+            forever. Together, we can save the oceans and help feed the world.
+            Visit <a href="http://oceana.org/">www.oceana.org</a> to learn more.
```
Comment (andresgnlez) [inline app/src/components/Partners/PartnerSection.jsx]: for links are pointing outside our site, use `target="_blank"`, please

**Comment by andresgnlez** [comment]
Have you optimized logo images/svgs? If not, please use `ImageOptim`  app to do it. Thanks!

**Comment by hectoruch** [comment]
Updates:

- Logos are optimazed
- Fix links targets
- Add slider on Research Partners section

**Pull request merged**